content/docs/user-guide/programming/remote-console.md

```diff
@@ -29,7 +29,7 @@ Universal Remote Console requires the use of a PC and works with both Android an
 
 1. After you update the `.cfg` file, deploy the game to mobile devices.
 
-   For more information, see [Deploy with O3DE Editor](/docs/userguide/mobile/android/build-deploy#run-the-deployment-tool) and [Creating a Release App](/docs/user-guide/platforms/ios/creating-release-app/).
+   For more information, see [Deploy with O3DE Editor](/docs/user-guide/mobile/android/build-deploy#run-the-deployment-tool) and [Creating a Release App](/docs/user-guide/platforms/ios/creating-release-app/).
```

Comment:
Even though this topic is in draft, let's get the intended target link corrected, or - in the case of iOS - what one might presume to be the future correct location.
```suggestion
   For more information, see [Android support for O3DE](/docs/user-guide/platforms/android/)<!-- and [iOS support for O3DE](/docs/user-guide/platforms/ios/)-->.
```